zgw/objecten-api/build.gradle

```diff
@@ -25,6 +25,8 @@ dependencies {
     implementation project(':plugin')
     implementation project(':contract')
     implementation project(':form')
+    implementation project(':openzaak')
```

Comment:
I'm not sure whether we should include openzaak. @ivo-ritense is this a problem?

Reply:
@Niek-Ritense please remove `openzaak` as dependency. Because its the module for the openzaak-connector and we're trying to deprecate that